### PR TITLE
[RHOAIENG-4852] update to colorful illustrations in project pages

### PR DESCRIPTION
**Pull request by nicolethoen**
Closes https://issues.redhat.com/browse/RHOAIENG-4852

## Description
Updates to the illustrations on the project UI screens as outlined by [RHOAI UX refresh for Summit 2024 doc](https://docs.google.com/document/d/1UwclZPg3jG891zqrv_Jkwo3IUzprIE0SYYlIJUAHR1M/edit) using the images in this [google drive folder](https://drive.google.com/drive/folders/10tW9AstzobKSs25D-QfCVnQew9l7yGZH).

## How Has This Been Tested?
Ran unit and cypress tests locally, also visually when running the dashboard locally

## Test Impact
N/A

## Request review criteria:

Self checklist (all need to be checked):
- [ ] The developer has manually tested the changes and verified that the changes work
- [ ] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has added tests or explained why testing cannot be added (unit or cypress tests for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

File: frontend/src/images/empty-state-cluster-storage.svg
Comment (andrewballantyne) [inline frontend/src/images/empty-state-cluster-storage.svg:1]: image

File: frontend/src/images/empty-state-data-connections.svg
Comment (andrewballantyne) [inline frontend/src/images/empty-state-data-connections.svg:1]: image

File: frontend/src/images/empty-state-model-serving.svg
Comment (andrewballantyne) [inline frontend/src/images/empty-state-model-serving.svg:1]: Might be for another PR -- but our Single & Multi Model Server doesn't have an infographic?
image
image

File: frontend/src/images/empty-state-pipelines.svg
Comment (andrewballantyne) [inline frontend/src/images/empty-state-pipelines.svg:1]: image

File: frontend/src/images/empty-state-projects-overview.svg
Comment (andrewballantyne) [inline frontend/src/images/empty-state-projects-overview.svg:1]: This one is interesting on space... based on how much space you take up -- it either looks weird or not.

Maybe this is another one that has too much white space?

Screenshot 2024-04-03 at 1 13 07 PM
Screenshot 2024-04-03 at 1 13 17 PM

**Review by andrewballantyne — CHANGES_REQUESTED** [review]
There seems to be a lot of whitespace at play here... is this what we want?

**Comment by nicolethoen** [comment]
@andrewballantyne I'll take the feedback back to the designer - maybe they can make larger images? Or at least speak to the white space.

**Comment by jgiardino** [comment]
@andrewballantyne regarding this comment:
> Might be for another PR -- but our Single & Multi Model Server doesn't have an infographic?
We currently don't have an image for those empty states. There is a separate jira to address all the variations of empty states for models: https://issues.redhat.com/browse/RHOAIENG-5083
You can see links to the design document in that jira. Any feedback on these designs are welcome in either that jira or figma file. 

Regarding the issue with image dimensions, I've followed up with the original designer to get help with exporting the images with the original dimensions. Thanks for pointing that out!

**Comment by nicolethoen** [comment]
Here are some screenshots with the updated images.

Screenshot 2024-04-04 at 5 04 15 PM
Screenshot 2024-04-04 at 5 04 44 PM
Screenshot 2024-04-04 at 5 05 12 PM
Screenshot 2024-04-04 at 5 05 30 PM

**Comment by jgiardino** [comment]
👍 Thanks for making these updates and including screencaps! I checked the deleted/added comparison under Files changed and the new illustrations look much more similar to the original.

**Pull request merged**